requirements/ci/py38.yml

```diff
@@ -11,8 +11,8 @@ dependencies:
 
 # Core dependencies.
   - cartopy >=0.20
-  - cf-units >=3
-  - cftime >=1.5
+  - cf-units ==3.0.1
```

Comment:
Bizarrely, although the `cf-units-3.0.1.post0` package is **actually** available on the `conda-forge` anaconda channel [here](https://anaconda.org/conda-forge/cf-units?version=3.1.0.post0&channel=main)... `conda` can't find it to install it! :confounded: 

However, there is no functional difference between `3.0.1` and `3.0.1.post0`, so it is safe to proceed with this verison pin "as-is" to get around this `conda` package version weirdness.